src/main/webapp/dev/js/main/adminAccountManagement.es6

```diff
@@ -136,7 +136,7 @@ function bindDeleteAccountAction() {
 
         const $clickedLink = $(event.target);
         const googleId = $clickedLink.data('googleId');
-        const existingCourses = document.getElementById(`courses_${googleId}`).innerHTML;
+        const existingCourses = $('#courses_${googleId}').innerHTML;
```

Comment:
Do not replace the existing apostrophes or backticks. They have different functions.